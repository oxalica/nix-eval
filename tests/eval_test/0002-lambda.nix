{
  a = (x: x) 1;
  b = (a: b: a) 1 (throw "wtf");
  c = (a: b: b) (throw "wtf") 2;
  d = (a: b: a + b) 1 2;
  e = (f: (x: f (x x)) (x: f (x x))) (f: x: if x == 0 then 1 else x * f (x - 1)) 10;
}
