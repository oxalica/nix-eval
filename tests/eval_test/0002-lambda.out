{
  a = <LAMBDA>;
  b = <LAMBDA>;
  c = <LAMBDA>;
  d = <LAMBDA>;
  e = <LAMBDA>;
}